redash/models/__init__.py

```diff
@@ -821,20 +821,42 @@ def evaluate(self):
     def subscribers(self):
         return User.query.join(AlertSubscription).filter(AlertSubscription.alert == self)
 
-    def render_template(self):
-        if not self.template:
+    def render_template(self, template=None):
```

Comment:
```suggestion
    def render_template(self, template):
```
Why do we need a default value for `template`?